pkg/extensions/_search.md

```diff
@@ -0,0 +1,194 @@
+`search` extension
```

Comment:
Should this file be here? I think there was a rebase issue.

Reply:
You're right, I will remove it right away.